apps/nextra/pages/en/build/smart-contracts/aptos-standards/fungible-asset.mdx

```diff
@@ -1,689 +1,310 @@
-struct does not have either key or store abilities, it can only be passed from
-one function to another but must be consumed by the end of a transaction.
-Specifically, it must be deposited back into a fungible store at the end of the
-transaction, which is defined as:
-
-```move
-module 0x1::fungible_asset {
-  #[resource_group_member(group = aptos_framework::object::ObjectGroup)]
-  struct FungibleStore has key {
-    /// The address of the base metadata object.
-    metadata: Object<Metadata>,
-    /// The balance of the fungible metadata.
-    balance: u64,
-    /// FAs transferring is a common operation, this allows for freezing/unfreezing accounts.
-    frozen: bool,
-  }
-}
-```
+1. `Object<Metadata>` - This represents details about the fungible asset itself, including information such as the `name`, `symbol`, and `decimals`.
+2. `Object<FungibleStore>` - This stores a count of fungible asset units owned by this account. Fungible assets are interchangeable with any other fungible asset that has the same metadata. An account *may* own more than one `FungibleStore` for a single Fungible Asset, but that is only for advanced use cases.
 
-<Callout type="info">
-  FAs are always stored in the top-level `FungibleStore` resource. This makes it
-  much easier to find, analyze, and control.
-</Callout>
+[This implementation](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-framework/sources/fungible_asset.move) is an improvement on the `coin` Standard because Move Objects are more customizable and extensible via smart contract. See the advanced guides on writing [Move Objects](aptos-object.mdx) for more details.
 
-The only extra field added here is `frozen`. if it is `true`, this object is
-frozen, i.e., deposit and withdraw are both disabled without using `TransferRef`
-in the next section.
+## Creating a new Fungible Asset (FA)
 
-### References
+At a high level, this is done by:
 
-_Reference_ (ref) is the means to implement granular permission control across
-different standards in Aptos. In different contexts, it may be called
-_capabilities_. The FA standard has three distinct refs for minting,
-transferring, and burning FA: `MintRef`, `TransferRef`, and `BurnRef`. Each ref
-contains a reference to the FA metadata:
+1. Creating a non-deletable Object to own the newly created Fungible Asset `Metadata`.
+2. Generating `Ref`s to enable any desired permissions.
+3. Minting Fungible Assets and transferring them to any account you want to.
 
-```move
-module 0x1::fungible_asset {
-  struct MintRef has drop, store {
-    metadata: Object<Metadata>
-  }
+To start with, the Fungible Asset standard is implemented using Move Objects. Objects by default are transferable, can own multiple resources, and can be customized via smart contract. For full details on Objects and how they work, please read [this guide](../learn-move/advanced-guides/objects.mdx). 
```

Comment:
I wonder if we can put this in Standards/objects so we don't have two objects pages.

Reply:
Talked about separately - definitely should merge those in a future PR :)